modules/percolator/src/main/java/org/elasticsearch/percolator/PercolateQueryBuilder.java

```diff
@@ -471,7 +471,7 @@ protected Query doToQuery(QueryShardContext context) throws IOException {
             docs.add(docMapper.parse(new SourceToParse(context.index().getName(), "_temp_id", document, documentXContentType)));
         }
 
-        FieldNameAnalyzer fieldNameAnalyzer = (FieldNameAnalyzer) docMapper.mappers().indexAnalyzer();
+        FieldNameAnalyzer fieldNameAnalyzer = (FieldNameAnalyzer)docMapper.mappers().indexAnalyzer();
```

Comment:
@romseygeek sorry this will give you conflicts in your PR, it is the outcome of splitting a bigger change into multiple PRs, and this is a leftover really. Easy to address the conflict though :)